kwalker314/wavepool-kristin-walker
# PR1: Fixed bug in news post teaser

added new teaser field in the database to store an editor-curated teaser string (because regexes to do that for us for every post sounds... really performance-intensive and HTML tags are infamously annoying to use regex for anyway)

to test:

1. go into the admin site and select any news post(s) you want to modify! it's your world, i'm just living in it
* you should consider modifying news posts that you will inspect on both the front page and archive page just to be thorough, so make sure at least one news post you add a teaser to has its "active" boolean set to true!
2. you should see a new "teaser" field that only supports plaintext (as full formatting with links, etc., can be limited to the article's dedicated page)
3. enter anything in the teaser you want for that news post~
4. go back to the main site, locate all the post(s) you modified, and confirm that their teaser shows up as it was written (both on the front page and archive page)
